data/aatifsohel.json

```diff
@@ -2,13 +2,28 @@
   "name": "Aatif Sohel",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Front-end Web Developer",
+  "bio": "Frontend Web Developer - Sharing my journey on Twitter",
   "avatar": "https://github.com/AatifSohel.png",
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "name": "Learn with me on Twitter",
       "url": "https://twitter.com/AatifSohel",
       "icon": "FaTwitter"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/aatifsohel",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/aatifsohel/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/aatifsohel/",
```

Comment:
This instagram link appears to lead to a 404 page. Maybe there is some typo you made?

Reply:
NO! I temporarily deactivated instagram account. That's why it is showing 404 page.